Source: Gaute Hope (c) <eg@gaute.vetsj.com>
Upstream: No
Reason: Set prefix, prepare for dynamic configuration by exheres
--- old/config.mk	2009-07-07 17:29:12.000000000 +0000
+++ work/config.mk	2009-10-14 08:41:37.000000000 +0000
@@ -3,7 +3,7 @@
 
 # Customize below to fit your system
 
-PREFIX = /usr/local
+PREFIX = /usr
 MANPREFIX = ${PREFIX}/share/man
 
 INCS = -I. -I/usr/include -I/usr/local/include 
@@ -13,10 +13,12 @@
 CFLAGS += -std=c99 -Os ${INCS} -DVERSION=\"${VERSION}\" -DNDEBUG
 LDFLAGS += -L/usr/lib -L/usr/local/lib ${LIBS}
 
+# gauteh: To be added by package
+#
 # Mouse handling
-CFLAGS += -DCONFIG_MOUSE
+#CFLAGS += -DCONFIG_MOUSE
 #CFLAGS += -DCONFIG_CMDFIFO
-CFLAGS += -DCONFIG_STATUSBAR
+#CFLAGS += -DCONFIG_STATUSBAR
 
 DEBUG_CFLAGS = ${CFLAGS} -UNDEBUG -O0 -g -ggdb -Wall
 
